OpenRA.Mods.Common/Traits/QuantizeFacingsFromSequence.cs

```diff
@@ -23,14 +23,13 @@ public class QuantizeFacingsFromSequenceInfo : ConditionalTraitInfo, IQuantizeBo
 		[Desc("Defines sequence to derive facings from.")]
 		public readonly string Sequence = "idle";
 
-		public int QuantizedBodyFacings(ActorInfo ai, SequenceProvider sequenceProvider, string race)
+		public int QuantizedBodyFacings(ActorInfo ai, SequenceProvider sequences, string faction)
 		{
 			if (string.IsNullOrEmpty(Sequence))
-				throw new InvalidOperationException("Actor " + ai.Name + " is missing sequence to quantize facings from.");
+				throw new InvalidOperationException($"Actor {ai.Name} is missing sequence to quantize facings from.");
 
 			var rsi = ai.TraitInfo<RenderSpritesInfo>();
-			var seq = sequenceProvider.GetSequence(rsi.GetImage(ai, race), Sequence);
-			return seq.InterpolatedFacings == -1 ? seq.Facings : seq.InterpolatedFacings;
+			return sequences.GetSequence(rsi.GetImage(ai, faction), Sequence).Facings;
```

Comment:
This is how it was previously and it did not work because `seq.InterpolatedFacings` needs to be returned if `InterpolatedFacings` are being used, not `Facings`. This is because `Facings` is never overridden by `InterpolatedFacings` since the interpolation is purely visual. so if you return `Facings` in all cases you are going to get the non-interpolated facings for all the sprites.

Reply:
This seems to have not changed in the latest commit.

Reply:
Interpolated vs "real" facings is an internal implementation detail that users don't need to know about!

`.Facings` returns the interpolated value if it is defined.